.buildkite/pipeline.yml

```diff
@@ -1,12 +1,11 @@
-# Nodes with values to reuse in the pipeline.
-common_params:
-  # Common plugin settings to use with the `plugins` key.
-  - &common_plugins
-    - automattic/a8c-ci-toolkit#3.3.0
-  # Common environment values to use with the `env` key.
-  - &common_env
-    # Be sure to also update the `.xcode-version` file when updating the Xcode image/version here
-    IMAGE_ID: xcode-15.4
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+agents:
+  queue: "mac"
```

Comment:
For this and the other instances of `"mac"`: I noticed in other repos that we haven't added quotation marks around `mac` ([PCiOS](https://github.com/Automattic/pocket-casts-ios/blob/trunk/.buildkite/pipeline.yml), [WPiOS](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/.buildkite/pipeline.yml)). I assume that adding them here wouldn't cause any issues and is probably a better practice than without quotes, but wanted to bring it up just in case.

Reply:
True... I think inlined strings in Yaml files don't really require the quotes. I think we're not really consistent there and I'm personally indifferent, but I think I got the pattern for WCiOS from WCAndroid 🤷

Reply:
I personally am a radical when it comes to `""` in YAML.

If they are not required (e.g. when a value includes `:` or starts with `*`) I don't use them. I feel it makes the YAML look leaner and easier to read.

But ☝️ I wouldn't enforce it as a rule unless we had a formatter that can apply it for us.

Reply:
Same as Gio, I feel like the YAML is more readable without the extra quotes if they are not necessary for YAML syntax disambiguation.

Besides, not wrapping the string in quotes means we can add quotes within the string if needed without having to escape them (e.g. `command: .buildkite/commands/run-my-script.sh "Hello world"` vs `command: ".buildkite/commands/run-my-script.sh \"Hello world\""`)

But same as Gio, while I like if we can be consistent across apps and files because it's always nicer to be consistent, I wouldn't be too strict about enforcing this particular rule either 🙃

Reply:
Agreed  🤝 @mokagio @AliSoftware. Moved from the _indifferent_ camp to the _if they're not required, don't use the quotes_ camp.

I'll send a small PR for WCiOS while at it and whenever I touch these files I'll remove unneeded quotes 👍